client/aliases.go

```diff
@@ -26,9 +26,10 @@ import (
 
 // aliasAction represents an action performed on aliases.
 type aliasAction struct {
-	Action  string   `json:"action"`
-	Snap    string   `json:"snap"`
-	Aliases []string `json:"aliases"`
+	Action string `json:"action"`
+	Snap   string `json:"snap,omitempty"`
+	Target string `json:"target,omitempty"`
```

Comment:
Per alias PR, the code seems to hint that this would be App. If so, this PR would need tweaking as well.